src/javascripts/Factories/treeViewFactory.js

```diff
@@ -27,8 +27,7 @@ ngapp.service('treeViewFactory', function(viewFactory) {
         if (view.class === 'record-view') {
             view.linkedTreeView = this;
             this.linkedRecordView = view;
-        }
-        if (view.class === 'referenced-by-view') {
+        } else if (view.class === 'referenced-by-view') {
```

Comment:
this code can also be removed, the tree view does not need to be able to link to the referenced by view.